### PR TITLE
Added receive and sent message updates

### PR DESCRIPTION
**Pull request by randy-girard**

### CODE REVIEW

**Comment by randy-girard** [comment]
@matthopson Here is my update to add in the received and sent messages. Does this look ok and did I miss anything? (Don't merge though yet)

File: src/persistence.coffee [hunk 1]
```diff
@@ -229,6 +229,18 @@ modifySDK = (sdk) ->
           err
       ).callback callback
 
+  # Reload the sent and received messages after saving
+  # a broadcast alert.
+  wrapMethod sdk, 'saveBroadcastAlert', (saveBroadcastAlert) ->
+    (broadcastAlert, callback) ->
+      saveBroadcastAlert.call(this, broadcastAlert, callback)
+      .then((result) ->
+        sdk.loadReceivedMessages({memberId: broadcastAlert.memberId})
```
Comment (matthopson) [inline src/persistence.coffee]: I would use:
```coffeescript
promises.when(
  sdk.loadReceivedMessages({memberId: broadcastAlert.memberId})
  sdk.loadSentMessages({memberId: broadcastAlert.memberId})
).then -> result```

**Comment by matthopson** [comment]
You will want to increment `pre` version on `package.json` and `teamsnap.coffee` - there was an update this morning, so it's currently on `pre-23`

**Comment by randy-girard** [comment]
@matthopson I have updated. Weird thing it, this only works for the first alert created. After that, they are not showing up.

**Comment by matthopson** [comment]
@randy-girard Are you sure you've rebuilt it? It sounds like the version you're using is still using the `wrapSave` method.

**Comment by randy-girard** [comment]
@matthopson This is what I have
```
  # Reload the sent and received messages after saving
  # a broadcast alert.
  wrapMethod sdk, 'saveBroadcastAlert', (saveBroadcastAlert) ->
    (broadcastAlert, callback) ->
      saveBroadcastAlert.call(this, broadcastAlert, callback)
      .then((result) ->
        promises.when(
          sdk.loadReceivedMessages({memberId: broadcastAlert.memberId})
          sdk.loadSentMessages({memberId: broadcastAlert.memberId})
        ).then -> result
      ).callback callback
```

**Comment by matthopson** [comment]
@randy-girard Try using:
```coffeescript
memberId = result.memberId
promises.when(
  sdk.loadReceivedMessages({memberId: memberId})
  sdk.loadSentMessages({memberId: memberId})
).then -> result

**Comment by randy-girard** [comment]
@matthopson: No dice.

**Pull request closed without merging**